modules/create-vm/pkg/utils/parse/clioptions_test.go

```diff
@@ -61,42 +45,6 @@ var _ = Describe("CLIOptions", func() {
 			Expect(results[0].Interface()).To(Equal(expectedValue))
 		}
 	},
-		Entry("returns valid defaults", &parse.CLIOptions{
```

Comment:
Keep this test with `VirtualMachineManifest` as input?

Reply:
it is part of Entry("handles vm cli arguments", &parse.CLIOptions{ test.
I updated this one to be focused on virtctl

Reply:
ACK